### DISCUSSION TITLE
sorting products in graphql queries.

### DISCUSSION THREAD

**Discussion by RZ-Jyothsna** [saleor/saleor#12175] [Q&A]
I need products to be fetched in priority order. I have added "priority" in attributes from saleor dashboard. How to sort products based on value of "priority" attribute.

**Comment by peelar** [comment]
Hello @RZ-Jyothsna 👋 

The [`products` query](https://docs.saleor.io/docs/3.x/api-reference/queries/products#code-style-fontweight-normal-productsbsortbybcodeproductorder-) has a `sortBy` argument that can accept [`attributeId`](https://docs.saleor.io/docs/3.x/api-reference/inputs/product-order#code-style-fontweight-normal-productorderbattributeidbcodeid-).
